sql/2019/06_Fonts/06_16.sql

```diff
@@ -0,0 +1,22 @@
+#standardSQL
+
+-- counts the local() inside @font-face
+
+WITH
+  response_bodies AS (
```

Comment:
I'd also wait to see the resolution of the CSS parser approach

Reply:
I made some changes on it using the parser approach